Doc/library/re.rst

```diff
@@ -1562,7 +1562,7 @@ find all of the adverbs in some text, they might use :func:`findall` in
 the following manner::
 
    >>> text = "He was carefully disguised but captured quickly by police."
```

Comment:
Suggest modifying the text to "The lying suspect was carefully disguised but captured quickly by police."

The addition of "lying" motivates the `\b` in the regex.   The use of "suspect" instead of "he" makes the sentence gender neutral.

Reply:
Thanks for the suggestion! Made the changes as suggested.

Reply:
One check still hasn't finished, what should I do?